swarmcg/optimize_model.py

```diff
@@ -40,6 +42,7 @@ def run(ns):
     #####################################
 
     # namespace variables not directly linked to arguments for plotting or for global package interpretation
+    # TODO: we should not generate new vars here, but before we go into the function
     ns.mismatch_order = False
     ns.row_x_scaling = True
     ns.row_y_scaling = True
```

Comment:
You will find that these are also defined in script evaluate_model.py, as they are used in the function compare_mdoels() which does the plotting.